Source/WebCore/html/HTMLProgressElement.cpp

```diff
@@ -70,11 +70,16 @@ bool HTMLProgressElement::childShouldCreateRenderer(const Node& child) const
 
 RenderProgress* HTMLProgressElement::renderProgress() const
 {
-    if (auto* renderProgress = dynamicDowncast<RenderProgress>(renderer()))
-        return renderProgress;
+    if (CheckedPtr renderProgress = dynamicDowncast<RenderProgress>(renderer()))
```

Comment:
Do we really get a static analysis warning here? This looks like unnecessary ref-count.

Reply:
Not really, I guess I shouldn't be doing this when it's for the return value

Reply:
I think we should do what static analysis asks, not more.